core/src/main/scala/cats/syntax/semigroupal.scala

```diff
@@ -23,13 +23,52 @@ package cats
 package syntax
 
 trait SemigroupalSyntax {
-  implicit final def catsSyntaxSemigroupal[F[_], A](fa: F[A])(implicit F: Semigroupal[F]): SemigroupalOps[F, A] =
+  @deprecated("Use `catsSntaxSemigroupalOps2`", "2.10.0")
+  final def catsSyntaxSemigroupal[F[_], A](fa: F[A])(implicit F: Semigroupal[F]): SemigroupalOps[F, A] =
```

Comment:
a minor thing, let's un-`implicit` the parameter there too:
```suggestion
  final def catsSyntaxSemigroupal[F[_], A](fa: F[A])(F: Semigroupal[F]): SemigroupalOps[F, A] =
```